source/frontend/src/features/documents/DocumentPreviewContainer.tsx

```diff
@@ -166,9 +166,7 @@ export const DocumentPreviewContainer: FC<
         pages={documentPages}
         resetDocumentPreview={resetDocumentPreview}
         handleDownload={() => {
-          downloadWrappedDocumentFileLatest(previewDocumentId).then(file =>
-            createFileDownload(file),
-          );
+          streamDocumentFileLatest(previewDocumentId).then(response => showFile(response));
```

Comment:
the preview should use the file streamer as well - or large file downloads will fail.